cts/cli/regression.crm_mon.exp

```diff
@@ -12,20 +12,20 @@ Node List:
   * GuestOnline: [ httpd-bundle-0 httpd-bundle-1 ]
 
 Active Resources:
-  * Clone Set: ping-clone [ping]:
+  * Clone Set: ping-clone [ping] :
```

Comment:
the expected outputs need to be updated again to match the current code (should fix CI issues)

Reply:
I've updated the PR, and as it is currently, I'm getting two cts-cli failures:
* Failed (rc=003): crm_simulate   - Show allocation scores with crm_simulate
* Failed (rc=003): crm_mon        - XML output grouped by node